test/dotnet-publish.Tests/Microsoft.DotNet.Tools.Publish.Tests.cs

```diff
@@ -217,38 +214,18 @@ public void CompilationFailedTest()
         [ActiveIssue(982)]
         public void PublishScriptsRun()
         {
-            // create unique directories in the 'temp' folder
-            var root = Temp.CreateDirectory();
-
-            var testAppDir = root.CreateDirectory("TestApp");
-            var testLibDir = root.CreateDirectory("TestLibrary");
+            TestScenario scenario = TestScenario.Create(Path.Combine(_testProjectsRoot, "TestAppWithLibrary"), skipRestore: true);
```

Comment:
I was looking for an example where we needed a scenario and not just an instance. Can you share the intent?